local-ros-backend.js

```diff
@@ -1,3 +1,3 @@
 module.exports.routes = {
-    '/api/ros/v0' : { host: 'http://localhost:8080' },
+    '/api/ros/v1' : { host: 'http://localhost:8080' },
```

Comment:
```suggestion
    '/api/ros/v1' : { host: 'http://localhost:8000' },
```

Reply:
Yes that line did need "v1.0", change pushed, also I believe we will need to raise and MR to update [here](https://github.com/RedHatInsights/cloud-services-config/blob/3bc11662bbc9570f19b0e6670e84b045a83ee41f/main.yml#L922)

Respectively, do we need to make changes on 3Scale?

Reply:
@saltgen  - After these changes, you will need to fix URL paths used inside templates - https://github.com/RedHatInsights/e2e-deploy/tree/master/templates/ros

Example - https://github.com/RedHatInsights/e2e-deploy/blob/master/templates/ros/ros-backend.yml#L107